packages/typespec-test/test/Fleet.Management/generated/typespec-ts/sdk/test/arm-test/src/classic/autoUpgradeProfileOperations/index.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ContainerServiceFleetContext } from "../../api/containerServiceFleetContext.js";
+import { AutoUpgradeProfileOperationsGenerateUpdateRunOptionalParams } from "../../api/autoUpgradeProfileOperations/options.js";
+import { generateUpdateRun } from "../../api/autoUpgradeProfileOperations/operations.js";
+import { PollerLike, OperationState } from "@azure/core-lro";
+
+/** Interface representing a AutoUpgradeProfileOperations operations. */
+export interface AutoUpgradeProfileOperationsOperations {
+  /** A long-running resource action. */
+  generateUpdateRun: (
+    resourceGroupName: string,
+    fleetName: string,
+    autoUpgradeProfileName: string,
+    options?: AutoUpgradeProfileOperationsGenerateUpdateRunOptionalParams,
+  ) => PollerLike<OperationState<void>, void>;
```

Comment:
issue here?

Reply:
The generated swagger([here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/stable/2025-03-01/fleets.json#L806-L848)) for this operation would be `GenerateResponse` but the lrometeadata for 


![image](https://github.com/user-attachments/assets/8268894f-bc60-4173-903f-2f06be283eeb)

![image](https://github.com/user-attachments/assets/e389fe32-313b-457f-a828-66cc413a41ed)